static/src/components/showcaseCards.js

```diff
@@ -21,33 +19,35 @@ const ShowcaseCard = styled.div`
   }
 `;
 
-const ShowcaseCards = () => {
-  const showcaseCards = useShowcase('all').map(
-    ({
-      node: {
-        data: { Name, Description, URL, ScreenShot, TeamMembers },
-      },
-    }) => (
-      <ShowcaseCard key={Name}>
-        <Img
```

Comment:
Would be nice to bring the images back in at some point

Reply:
* You don't need to act on this in any way, just want to keep this in mind going forward.